modules/tinydnsbackend/tinydnsbackend.cc

```diff
@@ -260,11 +260,11 @@ bool TinyDNSBackend::get(DNSResourceRecord &rr)
 			rr.auth = true;
 
 			rr.ttl = pr.get32BitInt();
-			uint64_t timestamp = pr.get32BitInt();
```

Comment:
This can't be a separate pull request.
Cassandra needs upgrade of libc6 and libc6-dev, this causes failure to tinydns due to namespace conflict between std::uint64_t and boost::uint64_t